public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -478,7 +478,7 @@ public SurveyQuestionResult apply(AchillesResult o) {
     }
 
     @Override
-    public ResponseEntity<DomainInfosAndSurveyModulesResponse> getDomainSearchResults(String query, String routeUrl){
+    public ResponseEntity<DomainInfosAndSurveyModulesResponse> getDomainSearchResults(String query){
```

Comment:
Not using these urls here anymore since we are tracking events by gtm now which by default tracks page url